### ISSUE TITLE
Add WarpInhibitor

### ISSUE THREAD

**Issue by TheNumenorean** [LemADEC/WarpDrive#17]
Prevents an area from being warped.

**Comment by LemADEC** [comment]
Bedrock does that already?

**Comment by TheNumenorean** [comment]
I mean a block that players can create, so that someone else cant just expand their warp field and warp their base away

**Comment by LemADEC** [comment]
Technically, they can already do that by creating a warp around their base. A WarpInhibitor remains a good idea for a 'cheaper' setup on a non-warping base.

**Comment by LemADEC** [comment]
By extension we could have a WarpInhibitionProjector that creates a bubble preventing warp in an area around it. A visual effect would shows the projector location so it's a valid target for those trying to escape from it.
Both inhibitors will need a 'warm up' to avoid abuses.

**Comment by lellasone** [comment]
Another valuable inhibitor might be one that prevented ships from warping into an area. (as a thought)

**Comment by LemADEC** [comment]
Yes, warp inhibitor should work both way. Eventually, it should intercept a warp if it's along the trajectory.

**Comment by LemADEC** [comment]
As discussed in #36, lellasone said:

> Warp Inhibition: 
> • Allowing players to prevent missiles from jumping inside their ship would help balance out tactical warp missiles. (perhaps there could be an EU cost equal to the cost of the jump blocked or some such)
> 
> • Making warp cores vulnerable to ICBM EMP’s would give players a robust and complex set of mechanics around trapping ships without making escape wholly impossible.
> 
> • Another direction would be to have a warp inhibitor that required setup (say 4 corners type thing) to make it viable against larger ships and bases but keep it from being the death knell to frigate/cruiser sized vehicles.

To sum up, we have either a block or a multiblock used to build a inhibition area.
Energy cost would be warmup, then idle, then per hit.

Overall, this looks a lot like a specialized forcefield, so I would propose to find/make a basic one and start from there.
